src/Stoa.ts

```diff
@@ -360,6 +361,31 @@ class Stoa extends WebService
         res.status(200).send();
     }
 
+    /**
+     * GET /block_height
+     *
+     * Return the highest block height stored in Stoa
+     */
+    private getBlockHeight (req: express.Request, res: express.Response)
+    {
+        logger.http(`GET /block_height`);
+
+        this.ledger_storage.getBlockHeight()
+            .then((row: any) => {
+                if (!row)
+                {
+                    res.status(400).send(`The block data not found.`);
+                    return;
+                }
+                res.status(200).send(JSON.stringify(row));
```

Comment:
Following my suggestion for LedgerStorage.getBlockHeight, I suggest:

```suggestion
            .then((row: Height | null) => {
                if (row == null)
                    res.status(400).send(`The block data not found.`);
                else
                    res.status(200).send(JSON.stringify(row));
```